sdk/keyvault/azure-security-keyvault-certificates/test/ut/certificate_client_test.cpp

```diff
@@ -888,3 +888,22 @@ TEST_F(KeyVaultCertificateClientTest, DISABLED_MergeCertificate)
     }
   }
 }
+
+TEST_F(KeyVaultCertificateClientTest, ServiceVersion)
+{
+  auto credential
+      = std::make_shared<Azure::Identity::ClientSecretCredential>("tenantID", "AppId", "SecretId");
+  {
+    // 7.3
+    EXPECT_NO_THROW(auto options = CertificateClientOptions(); CertificateClient certificateClient(
+                        "http://account.vault.azure.net", credential, options);
+                    EXPECT_EQ(options.Version, "7.3"););
```

Comment:
Line 900 seems to be confirming that the compiler correctly generated code. 

Do you have reason to believe that the compiler will incorrectly process the initializer for Version?

Reply:
JUST TO MAKE SURE,